tensorrt_llm/_torch/modules/mamba/gdn_mixer.py

```
@@ -0,0 +1,910 @@
+        blk_b_ptr = mixed_ba + i_bs * NUM_HEADS_QK * BA_DIM_T + i_qk * BA_DIM_T + i
+        blk_b_st_ptr = b + i_bs * NUM_HEADS_V + i_qk * NUM_HEADS_V // NUM_HEADS_QK + i
+        tl.store(blk_b_st_ptr, tl.load(blk_b_ptr))
+    for i in tl.static_range(b_end, a_end):
+        blk_a_ptr = mixed_ba + i_bs * NUM_HEADS_QK * BA_DIM_T + i_qk * BA_DIM_T + i
+        blk_a_st_ptr = (a + i_bs * NUM_HEADS_V +
+                        i_qk * NUM_HEADS_V // NUM_HEADS_QK + (i - b_end))
+        tl.store(blk_a_st_ptr, tl.load(blk_a_ptr))
+
+
+@triton.jit
+def fused_qkvzba_split_reshape_cat_kernel(
```

Flake8 (7.3.0)

[error] redefinition of unused 'fused_qkvzba_split_reshape_cat_kernel' from line 49

(F811)